mixer/pkg/api/grpcServer.go

```diff
@@ -129,6 +129,7 @@ func (s *grpcServer) Check(legacyCtx legacyContext.Context, req *mixerpb.CheckRe
 	// compatReqBag ensures that preprocessor input handles deprecated attributes gracefully.
 	compatReqBag := &compatBag{requestBag}
 	preprocResponseBag := attribute.GetMutableBag(nil)
+	addContextAttributes(preprocResponseBag)
```

Comment:
Looks like context.time is not available during preprocess. It is stuffed directly into preprocresponse bag.
Perhaps add a childbag of compareqbag with this attribute and pass it as an input to preprocess.

Reply:
This is becoming really awkward. I've updated the code to show structure, but I really am not happy with where this is heading. We really should need 3 bags at most:
- The proto bag
- The compatibility bag
- The mutable bag for generated/context attributes.

I'll take another stab at simplifying the overall bag juggling.

Reply:
Lets drop the compatibility bag, Though it is a good insertion points when we have incompatibilities in the future.

At the very least, the compatibility should be a no-op now. It should not do any  alias lookups, any more.